rx-extensions/src/main/java/com/appunite/rx/functions/Functions1.java

```diff
@@ -129,4 +142,38 @@ public String call(CharSequence charSequence) {
             }
         };
     }
+
+    @Nonnull
+    public static Func1<CharSequence, Boolean> isEmpty() {
+        return new Func1<CharSequence, Boolean>() {
+            @Override
+            public Boolean call(CharSequence charSequence) {
+                return charSequence.length() == 0;
+            }
+        };
+    }
+
+    /**
+     * Checks if propagated value is equal to any of passed values.
+     * @param values values which are compared to propagated value.
+     * @return true if propagated value is equal to at least one of passed values, false otherwise.
+     * If values length is equal to 0 or passed null reference this method returns false.
+     */
+    @Nonnull
+    public static Func1<? super Object, Boolean> isEqualTo(@Nullable final Object... values) {
```

Comment:
Is it array usefull?
I suggest to add @Nonnull
checkArgument(values != null, "Values can not be null");
checkArgument(values.length > 0, "You need to speciffy at least one object");